manifest.yml

```diff
@@ -10,12 +10,13 @@ applications:
     - rm-redis
     - rm-rabbitmq
   env:
-    internetAccessCodeSvc_connectionConfig_host: iacsvc.apps.devtest.onsclofo.uk
```

Comment:
Do we want to leave this on devtest rather than perf?

Reply:
I'll reset them to master